releasetasks/templates/firefox/beetmove_image.yml.tmpl

```diff
@@ -0,0 +1,57 @@
+# keep first line
+-
+    taskId: "{{ stableSlugId("beetmove_image") }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: taskcluster-images
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        routes:
+            - tc-treeherder-stage.{{ branch }}.{{ revision_hash }}
+            - index.releases.v1.{{ branch }}.{{ revision }}.{{ product }}.{{ version | replace(".", "_") }}.build{{ buildNumber }}.beetmove_image
+            - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.beetmove_image
+        payload:
+            artifacts:
+                public/image.tar:
+                      expires: {{ now.replace(days=365) }}
+                      path: /artifacts/image.tar
+                      type: file
+            features:
+                dind: true
```

Comment:
It enabled the "dind" feature (Docker in Docker) of docker-worker, so the script can run `docker build/save`. See https://github.com/taskcluster/docker-worker/blob/master/lib/features/dind.js, https://github.com/taskcluster/dind-service, https://dxr.mozilla.org/mozilla-central/source/testing/docker/image_builder/bin/build_image.sh#34